custom_model_runner/datarobot_drum/resource/predict_mixin.py

```diff
@@ -107,7 +107,7 @@ def _check_mimetype_support(self, mimetype):
             return {"message": error_message}, HTTP_422_UNPROCESSABLE_ENTITY
         return None
 
-    def _predict(self, logger=None):
+    def _do_predict(self, logger=None):
```

Comment:
having two functions named do_predict and _do_predict seems confusing. Can you give either of them a more descriptive name?

Reply:
Generally I don't see this confusing as `some_func` is usually a public wrapper to `_some_func`.
But here I tried to rename as we had `do_predict` in BasePredictor as well.

Reply:
to me, `do_predict` + `_do_predict` is much better than `do_predict` + `do_predict_predictor`. method name is super confusing.

`some_func` + `_some_func` is a common pattern.